pages/dkp/konvoy/2.0/install/advanced/cluster_autoscaler/index.md

```diff
@@ -36,13 +36,6 @@ Complete the following items before continuing with the Cluster Autoscaler:
 
 ### Run Cluster Autoscaler
 
-Apply these annotations to the MachineDeployment to enable the Cluster Autoscaler.
```

Comment:
why did we take this out?

Reply:
bc of deepak's review https://github.com/mesosphere/konvoy2/pull/416#discussion_r667238723